### DISCUSSION TITLE
How diable automaticly plugin synchronizing while start nvim (with lazy.nvim)?

### DISCUSSION THREAD

**Discussion by lainegates** [LazyVim/LazyVim#2375] [Q&A] [answered]
LazyNvim has an excellent plugin management ui. 
And it seems that nvim will try to update all plugins when starting every-time .

For my environment, updating nvim plugins is a troublesome task; it requires turning on a VPN to accomplish.
Thus, I want to update plugins by myself, e.g. once a week when I have time.
So, Is there a way to disable the automatic plugins synchronizing ?

**Comment by dpetka2001** [comment, marked as answer]
In your `~/.config/nvim/lua/config/lazy.lua` you should change, if they already exist or add otherwise, the following setting
```lua
-- ~/.config/nvim/lua/config/lazy.lua
require("lazy").setup({
  -- some options
  checker = { enabled = false },
  -- more options
})
```
You can check the default options of [lazy.nvim](https://github.com/folke/lazy.nvim#%EF%B8%8F-configuration) for more.
